GameServer/packets/Server/PacketLib1126.cs

```diff
@@ -340,10 +348,10 @@ public override void SendUpdateWeaponAndArmorStats()
 				pak.WriteByte(0x00); //unk
 
 				// weapondamage
-				var wd = (int)(m_gameClient.Player.WeaponDamage(m_gameClient.Player.AttackWeapon) * 100.0);
-				pak.WriteByte((byte)(wd / 100));
+				var wd = (int)(m_gameClient.Player.WeaponDamage(m_gameClient.Player.AttackWeapon) * 100);
+				pak.WriteByte((byte)(wd / 0x100));
```

Comment:
`0x100` ? Hexa ?

Reply:
Yes, it is to split the "short" value into separate bytes.

Reply:
In this case use bit operations instead of a costly division (shift + AND)

Reply:
Division takes one cycle on almost any CPU (latency is higher tho), but even if it would take 50 cycles it would make no perceivable difference, because it is called rather seldom. For readability I chose division/modulo over `>>`/`&`.

Reply:
I think there is no point of optimization here, the compiler should have no issue to make that as fast as a bitwise operation.

Just keep the most readable version since it's about bits (keeping the high part of the int16), i find bitwise operations more relevant but it's just my opinion.

Reply:
Agree this kind of optimization is normally applied by the compiler whatever form it takes.